front_end/src/components/Accounts/AccountForm/AccountForm.js

```diff
@@ -50,30 +65,61 @@ export default class AccountForm extends Component {
               <div className="col-xs-4">
                 <div className="form-group">
                   <label htmlFor="AccountContactName">Contact Name</label>
-                  <input className="form-control" type="text" id="AccountContactName" name="contact_name" ref="contact_name" defaultValue={data.contact_name} />
+                  <input className="form-control" type="text" id="AccountContactName" name="contact_name" ref="contact_name" defaultValue={data.contact_name} onChange={this.handleChange}/>
                 </div>
               </div>
               <div className="col-xs-4">
                 <div className="form-group">
                   <label htmlFor="AccountContactEmail">Contact Email</label>
-                  <input className="form-control" type="text" id="AccountContactEmail" name="contact_email" ref="contact_email" defaultValue={data.contact_email} data-parsley-type="email"/>
+                  <input className="form-control" type="text" id="AccountContactEmail" name="contact_email" ref="contact_email" defaultValue={data.contact_email} onChange={this.handleChange} data-parsley-type="email"/>
                 </div>
               </div>
               <div className="col-xs-4">
                 <div className="form-group">
                   <label htmlFor="AccountContactPhone">Contact Phone</label>
-                  <input className="form-control" type="text" id="AccountContactPhone" name="contact_phone" ref="contact_phone" defaultValue={data.contact_phone} />
+                  <input className="form-control" type="text" id="AccountContactPhone" name="contact_phone" ref="contact_phone" defaultValue={data.contact_phone} onChange={this.handleChange}/>
+                </div>
+              </div>
+            </div>
+            <div className="disabled">
```

Comment:
why are all these disabled? any way to enable them? oh, I am guessing they are placeholders until we get support for them in API.